pypeit/core/wavecal/spectrographs/templ_ldt_deveny.py

```diff
@@ -0,0 +1,146 @@
+""" Generate the wavelength templates for LDT/DeVeny"""
```

Comment:
Thanks for this!

Reply:
Input templates being included in pypeit/PypeIt-development-suite#203.